src/App/App.js

```diff
@@ -1,15 +1,14 @@
 import React, { Component } from 'react';
 import PlayerCardList from '../Player/PlayerCardList';
-import TurnTimer from '../TurnTimer/TurnTimer';
+import { playersExample } from '../Player/playerHelpers';
```

Comment:
Soon we'll wire this up to a basic web server, and we'll get rid of the mock data here.